launcher/src/main.rs

```diff
@@ -184,6 +184,23 @@ struct Args {
     #[clap(long, env, value_enum)]
     dtype: Option<Dtype>,
 
+    /// Data type for kv cache storage. If "auto", will use model
+    /// data type. FP8_E5M2 (without scaling) is only supported on cuda
+    /// version greater than 11.8. On ROCm (AMD GPU), FP8_E4M3 is instead
+    /// supported for common inference criteria.
+    #[clap(default_value = "auto", long, env)]
+    kv_cache_dtype: Option<String>,
+
+    /// Path to the JSON file containing the KV cache
+    /// scaling factors. This should generally be supplied, when
+    /// KV cache dtype is FP8. Otherwise, KV cache scaling factors
+    /// default to 1.0, which may cause accuracy issues.
+    /// FP8_E5M2 (without scaling) is only supported on cuda version
+    /// greater than 11.8. On ROCm (AMD GPU), FP8_E4M3 is instead
+    /// supported for common inference criteria.
+    #[clap(long, env)]
+    quantization_param_path: Option<String>,
```

Comment:
(I'm trying to avoid adding too many flags, TGI already has too many, and since we don't break, we never remove stuff that was added, that's why if we can read the information from some consistant config in the repo it keeps the interface for the user simpler)